app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentInit.kt

```diff
@@ -17,6 +17,8 @@ class ScheduleAppointmentInit : Init<ScheduleAppointmentModel, ScheduleAppointme
       effects.add(LoadAppointmentFacilities(model.patientUuid))
     }
 
+    effects.add(LoadTeleconsultRecord(model.patientUuid))
```

Comment:
If the `TeleconsultRecord` is already present, we don't have to load it again. That would be useful to avoid loading it again during state restoration.